### ISSUE TITLE
manual/javaGuide/main/sql/JavaJPA.md

### ISSUE THREAD

**Issue by garbagetown** [playframework-ja/translation-project#569]
- https://github.com/garbagetown/playdocja/blob/2.2.0/documentation/2.2.0/manual/javaGuide/main/sql/JavaJPA.md

```
--- /Users/garbagetown/Desktop/2.1.5/manual/javaGuide/main/sql/JavaJPA.md   2013-09-19 22:53:02.000000000 +0900
+++ /Users/garbagetown/Desktop/2.2.0/manual/javaGuide/main/sql/JavaJPA.md   2013-09-19 10:11:22.000000000 +0900
@@ -1,23 +1,26 @@
 # Integrating with JPA

-## Exposing the datasource through JNDI
+## Adding dependencies to your project

-JPA requires the datasource to be accessible via JNDI. You can expose any Play-managed datasource via JDNI by adding this configuration in `conf/application.conf`:
+First you need to tell play that your project need javaJpa plugin which provide JDBC and JPA api dependencies.
+
+There is no built-in JPA implementation in Play; you can choose any available implementation. For example, to use Hibernate, just add the dependency to your project:

 '''
-db.default.driver=org.h2.Driver
-db.default.url="jdbc:h2:mem:play"
-db.default.jndiName=DefaultDS
+val appDependencies = Seq(
+  javaJpa,
+  "org.hibernate" % "hibernate-entitymanager" % "3.6.9.Final" // replace by your jpa implementation
+)
 '''

-## Adding a JPA implementation to your project
+## Exposing the datasource through JNDI

-There is no built-in JPA implementation in Play 2.0; you can choose any available implementation. For example, to use Hibernate, just add the dependency to your project:
+JPA requires the datasource to be accessible via JNDI. You can expose any Play-managed datasource via JDNI by adding this configuration in `conf/application.conf`:

 '''
-val appDependencies = Seq(
-  "org.hibernate" % "hibernate-entitymanager" % "3.6.9.Final"
-)
+db.default.driver=org.h2.Driver
+db.default.url="jdbc:h2:mem:play"
+db.default.jndiName=DefaultDS
 '''

 ## Creating a persistence unit
@@ -31,7 +34,7 @@
              xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
              xsi:schemaLocation="http://java.sun.com/xml/ns/persistence http://java.sun.com/xml/ns/persistence/persistence_2_0.xsd"
              version="2.0">
-             
+
     <persistence-unit name="defaultPersistenceUnit" transaction-type="RESOURCE_LOCAL">
         <provider>org.hibernate.ejb.HibernatePersistence</provider>
         <non-jta-data-source>DefaultDS</non-jta-data-source>
@@ -39,10 +42,16 @@
             <property name="hibernate.dialect" value="org.hibernate.dialect.H2Dialect"/>
         </properties>
     </persistence-unit>
-    
+
 </persistence>
 '''

+Finally you have to tell Play, which persistent unit should be used by your JPA provider. This is done by the `jpa.default` property in your `application.conf`.
+
+'''
+jpa.default=defaultPerstistenceUnit
+'''
+
 ## Annotating JPA actions with `@Transactional`

 Every JPA call must be done in a transaction so, to enable JPA for a particular action, annotate it with `@play.db.jpa.Transactional`. This will compose your action method with a JPA `Action` that manages the transaction for you:
@@ -74,4 +83,3 @@
 '''

 > **Next:** [[Using the cache | JavaCache]]
-

```

**Comment by garbagetown** [comment]
pull reqed at #677

**Comment by wm3** [comment]
merged #677
